.circleci/config.yml

```diff
@@ -25,32 +25,45 @@ jobs:
       - checkout
       - run:
           name: Configure Git
-          command: | 
+          command: |
             git config --global user.name "Bodywork Machine Learning Ltd"
             git config --global user.email "info@bodyworkml.com"
       - run:
           name: Installing Python dependencies
           command: pip install -r requirements_dev.txt
       - run: 
           name: Running tests
-          command: tox -e py38-unit_and_functional_tests,py38-static_code_analysis -- -s -vv
+          command: tox -e py38-unit_and_functional_tests,py38-static_code_analysis -- -s -vv --junitxml=~/tox/junit.xml
```

Comment:
Why do you need to use `--junitxml` to create test results and why are you switching to the output path afterwards?

Reply:
For CircleCI to generate the test summary (the tests tab in the build).